lib/guardian.ex

```diff
@@ -724,16 +713,20 @@ defmodule Guardian do
   end
 
   @doc """
-  Provides a way to track an invalid return tuple via a nicer reason
+  Ensures the return result of calling `{mod, func, args}`
+  is a 2 element tuple with `{:ok, any}`  or `{:error, any}`
+  If not, it will return an error with a message detailing
+  the module, function and return that was received
   """
 
-  @spec validate_conditional_tuple({:ok, any} | {:error, any} | any, {module, atom}) :: {:ok, any} | {:error, any}
-  def validate_conditional_tuple({:ok, _} = resp, _),
-    do: resp
-  def validate_conditional_tuple({:error, _} = resp, _),
-    do: resp
-  def validate_conditional_tuple(resp, {m, f}),
-   do: {:error, "Invalid return for #{m}##{f} - #{inspect(resp)}"}
+  def returning_tuple({mod, func, args}) do
```

Comment:
I need to see this used in context of userland code to understand why it's needed to be public. If it's necessary to export then maybe a new name would fit better?